_posts/2017/2017-05-09-how-to-successfully-run-a-tech-blog.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: "How to successfully run a company tech blog"
+date: "2017-05-09 10:00:00"
+image: blog-header/social-like.jpg
+categories: tech-stories
+authors: ["Rose"]
+---
+
+Since we develop e-commerce solutions at [ePages](https://www.epages.com/us/), there were enough techies willing to help us out when we first decided to set up a [company tech blog](https://developer.epages.com/blog) in spring 2015.
```

Comment:
At [ePages](https://www.epages.com/us/) we develop e-commerce solutions. Thus, we had enough techies around that supported us setting up a [company tech blog](https://developer.epages.com/blog) in spring 2015.